sdk/storage/azure-storage-files-shares/inc/azure/storage/files/shares/share_file_attribute.hpp

```diff
@@ -5,207 +5,213 @@
 
 namespace Azure { namespace Storage { namespace Files { namespace Shares {
 
-  enum class FileAttributes
-  {
-    /**
-     * @brief The File or Directory is read-only.
-     */
-    ReadOnly = 1,
-
-    /**
-     * @brief The File or Directory is hidden, and thus is not included in an ordinary directory
-     * listing.
-     */
-    Hidden = 2,
-
-    /**
-     * @brief The File or Directory is a systemfile.  That is, the file is part of the operating
-     * system or is used exclusively by the operating system.
-     */
-    System = 4,
-
-    /**
-     * @brief The file  or directory is a standard file that has no special attributes. This
-     * attribute is valid only if it is used alone.
-     */
-    None = 8,
-
-    /**
-     * @brief The file is a directory.
-     */
-    Directory = 16,
-
-    /**
-     * @brief The file is a candidate for backup or removal.
-     */
-    Archive = 32,
-
-    /**
-     * @brief The file or directory is temporary. A temporary file contains data that is needed
-     * while an application is executing but is not needed after the application is finished. File
-     * systems try to keep all the data in memory for quicker access rather than flushing the data
-     * back to mass storage. A temporary file should be deleted by the application as soon as it is
-     * no longer needed.
-     */
-    Temporary = 64,
-
-    /**
-     * @brief The file or directory is offline. The data of the file is not immediately available.
-     */
-    Offline = 128,
-
-    /**
-     * @brief The file or directory will not be indexed by the operating system's content indexing
-     * service.
-     */
-    NotContentIndexed = 256,
-
-    /**
-     * @brief The file or directory is excluded from the data integrity scan. When this value is
-     * applied to a directory, by default, all new files and subdirectories within that directory
-     * are excluded from data integrity.
-     */
-    NoScrubData = 512
-  };
-
-  inline FileAttributes operator|(FileAttributes lhs, FileAttributes rhs)
-  {
-    using type = std::underlying_type_t<FileAttributes>;
-    return static_cast<FileAttributes>(static_cast<type>(lhs) | static_cast<type>(rhs));
-  }
-
-  inline FileAttributes& operator|=(FileAttributes& lhs, FileAttributes rhs)
-  {
-    lhs = lhs | rhs;
-    return lhs;
-  }
-
-  inline FileAttributes operator&(FileAttributes lhs, FileAttributes rhs)
-  {
-    using type = std::underlying_type_t<FileAttributes>;
-    return static_cast<FileAttributes>(static_cast<type>(lhs) & static_cast<type>(rhs));
-  }
-
-  inline FileAttributes& operator&=(FileAttributes& lhs, FileAttributes rhs)
-  {
-    lhs = lhs & rhs;
-    return lhs;
-  }
-
-  inline FileAttributes FileAttributesFromString(const std::string& fileAttributesString)
-  {
-    FileAttributes result = static_cast<FileAttributes>(0);
-
-    if (fileAttributesString == "ReadOnly")
+  namespace Models {
```

Comment:
Are you sure you've re-formatted with clang-format? It seems to me that this `namespace Models {` line should come in the same line as other namespaces

Reply:
It shouldn't be, there are `Models` and `Details` namespace within this file under namespace `Azure::Storage::Files::Shares`, that will new up a line for `Models` or `Details` namespace.